templates/about_us/index.html.twig

```diff
@@ -0,0 +1,65 @@
+{% extends 'base.html.twig' %} {% block title %} {{ parent() }}
+{{ pageName }} {% endblock %} {% block stylesheet %}
+<link rel="stylesheet" href="assets/styles/pages/about.css" />
+{% endblock %} {% block main %}
+<div class="apropos-page section">
```

Comment:
sorry for the misscomprehension, rename this div into main, since you don't have a main tag wrapping your body block in your base.html.twig